ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -2390,13 +2425,14 @@ private enum UnityLayers {
       try {
         ConfigNode initial_states = GameDatabase.Instance.GetConfigs(
             principia_initial_state_config_name_)[0].config;
-        plugin_ =
-            Interface.NewPlugin(initial_states.GetValue("game_epoch"),
-                                initial_states.GetValue("solar_system_epoch"),
-                                Planetarium.InverseRotAngle);
```

Comment:
Why the changes?  I preferred it this way.

Reply:
Unique